### PR TITLE
Create ChartWidgetEmptyState component

### PR DESCRIPTION
**Pull request by sauldom102**
Screenshot 2024-11-11 at 12 21 48

### CODE REVIEW

File: lib/experimental/Widgets/ChartWidgetEmptyState/index.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  content: string
+  buttonLabel?: string
+  buttonIcon?: IconType
+  buttonAction?: () => void
+  type: Type
+}
+
+const ROOT_CLASSNAMES: Record<Type, string> = {
+  "line-chart": "border-f1-border-accent-alpha20",
+  "bar-chart": "border-f1-border-promoteAlpha30",
```
Comment (nlopin) [inline lib/experimental/Widgets/ChartWidgetEmptyState/index.tsx:22]: is it correct? the other classnames follow different naming pattern

File: lib/experimental/Widgets/ChartWidgetEmptyState/index.tsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    return (
+      <Card
+        className={cn(
+          "relative flex gap-4 overflow-hidden border-dashed",
+          rootClassName
+        )}
+        ref={ref}
+      >
+        <CardHeader className="-mr-1 -mt-1">
+          <CardTitle className="truncate">{title}</CardTitle>
```
Comment (nlopin) [inline lib/experimental/Widgets/ChartWidgetEmptyState/index.tsx:59]: I would never truncate the title, or at least not on the first line

File: lib/experimental/Widgets/ChartWidgetEmptyState/index.tsx [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+    return (
+      <Card
+        className={cn(
+          "relative flex gap-4 overflow-hidden border-dashed",
+          rootClassName
+        )}
+        ref={ref}
+      >
+        <CardHeader className="-mr-1 -mt-1">
```
Comment (nlopin) [inline lib/experimental/Widgets/ChartWidgetEmptyState/index.tsx:58]: is it necessary? Maybe we should reconsider the card style?

File: lib/experimental/Widgets/ChartWidgetEmptyState/index.tsx (hunk 1 above)
Comment (sauldom102) [reply lib/experimental/Widgets/ChartWidgetEmptyState/index.tsx]: Fixed it!

File: lib/experimental/Widgets/ChartWidgetEmptyState/index.tsx (hunk 3 above)
Comment (sauldom102) [reply lib/experimental/Widgets/ChartWidgetEmptyState/index.tsx]: Yep, some adjustment is needed, according to figma the card at the top and on the right has less spacing

**Review by nlopin — APPROVED** [review]

**Pull request merged**